cloud/deploy/deploy.go

```diff
@@ -176,28 +188,28 @@ func Deploy(path, runtimeID, wsID, pytest, envFile, imageName, deploymentName st
 		return nil
 	}
 
-	deployInfo, err := getDeploymentInfo(runtimeID, wsID, deploymentName, prompt, domain, client)
+	deployInfo, err := getDeploymentInfo(deployInput.RuntimeID, deployInput.WsID, deployInput.DeploymentName, deployInput.Prompt, domain, client)
 	if err != nil {
 		return err
 	}
 
 	deploymentURL := "cloud." + domain + "/" + deployInfo.workspaceID + "/deployments/" + deployInfo.deploymentID + "/analytics"
```

Comment:
I may have an example of doing away with this. Leaving this comment as a reminder to discuss it.

Reply:
We  discussed this and are tracking it separately in #744 so resolving this.